src/game/carrier.c

```diff
@@ -0,0 +1,213 @@
+#include "game/carrier.h"
+
+#include "game/gamebuf.h"
+#include "game/gameflow.h"
+#include "game/inventory.h"
+#include "game/items.h"
+#include "global/const.h"
+#include "global/types.h"
+#include "global/vars.h"
+#include "log.h"
+
+#include <stdbool.h>
+#include <stddef.h>
+
+#define AMMO_OFFSET 4
```

Comment:
Can we do without this const?

Reply:
Perhaps an explicit switch statement to convert gun to ammo would be better, or a map even.

Reply:
Yes that's exactly what I had on my mind.